tests/core/ssl/test_ssl.py

```diff
@@ -23,7 +21,7 @@
 from tests.util.socket import find_available_listen_port
 
 
-async def establish_connection(server: ChiaServer, ssl_context) -> bool:
+async def establish_connection(server: ChiaServer, self_hostname: str, ssl_context) -> bool:
```

Comment:
I don't see how we can get away without passing in `self_hostname` to `establish_connection` - it is used twice in the body. `self_hostname` just used to be a global that silently brought state into every test method in a module that imported it.

Reply:
they way we do in `main` right now. Import it.

Reply:
I'm suggesting you revert this part of your patch. to import `self_hostname` in all places it's used, rather than importing it in *some* places and then pass it in as a new parameter